crates/bin/pindexer/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub use cometindex::Indexer;
```

Comment:
this should also re-export the `AppView` trait i think, to avoid needing to depend on both crates

Reply:
great idea! b33cf04 adds this :hammer: